posthog-android/src/test/java/com/posthog/android/PostHogFake.kt

```diff
@@ -22,7 +22,7 @@ public class PostHogFake : PostHogInterface {
         properties: Map<String, Any>?,
         userProperties: Map<String, Any>?,
         userPropertiesSetOnce: Map<String, Any>?,
-        groupProperties: Map<String, Any>?,
+        groups: Map<String, Any>?,
```

Comment:
I wonder if this should have been `Map<String, String>` since `group(type: String, key: String)` are Strings only